releases/nightly-build/compose-files/README.md

```diff
@@ -4,7 +4,7 @@ This folder contains the docker compose files that pull and run the EdgeX images
 
```

Comment:
I found the multiple README.md files confusing. I would prefer the compose builder README.md documents everything and remove this one.

Reply:
The READMEs are not the same. 
This one is targeted to users. 
The one in compose-builder is targeted to developers and those users creating custom compose files.